common/stats/stats_names.go

```diff
@@ -353,6 +353,12 @@ const (
 	*/
 	WorkerActiveRunsGauge = "activeRunsGauge"
 
+	/*
+		The disk size change for the indicated directory seen when running the task.
+		The stat will be qualified with a logical name for the directory
```

Comment:
Qualified how - commandDirUsageKb_X? usageKbX? Typically we use _ to delimit these types of stats.